packages/slice-machine/components/Simulator/components/Toolbar/ScreensizeInput/index.tsx

```diff
@@ -0,0 +1,94 @@
+import { ScreenDimensions } from "@lib/models/common/Screenshots";
+import { Flex, Input, Label, ThemeUIStyleObject } from "theme-ui";
+
+export enum ScreenSizeOptions {
+  DESKTOP = "Desktop",
+  TABLET = "Tablet",
+  MOBILE = "Mobile",
+  CUSTOM = "Custom",
+}
+
+export const ScreenSizes: Record<string, ScreenDimensions> = {
+  [ScreenSizeOptions.DESKTOP]: {
+    width: 1200,
```

Comment:
Where do those values come from ?
Breakpoints always seems so random on multiple websites that i'm wondering where yours come from 😄